sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -58,7 +58,7 @@ az cognitiveservices account create \
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-ai-textanalytics</artifactId>
-    <version>5.0.4</version>
+    <version>5.0.5</version>
```

Comment:
Didn't we decide to update this to the version that we are releasing and in the Increment PR we change it back to stable ?